package.json

```diff
@@ -17,6 +17,8 @@
     "prepare": "tsdx build --tsconfig tsconfig.json --entry src/index.ts"
   },
   "devDependencies": {
+    "@ionio-lang/ionio": "^0.3.0",
```

Comment:
install the whole ionio just for types does not seems great. for now copy/paste the interefaces should be fine

Reply:
You only need it while developing but `npm install marina-provider` does not fetch ionio sources thanks to `import type` (removed once TS is compiled).